eemeter/caltrack/usage_per_day.py

```diff
@@ -185,6 +185,41 @@ def _json_or_none(obj):
             data["candidates"] = [candidate.json() for candidate in self.candidates]
         return data
 
+    @classmethod
+    def fromJson(cls, data):
```

Comment:
Code style - can you change this from `fromJson` to `from_json` to keep the consistency with the PEP 8 recommendation and the rest of the code? See https://www.python.org/dev/peps/pep-0008/#function-and-variable-names